ui/media/js/main.js

```diff
@@ -839,7 +849,10 @@ function createTask(task) {
     let taskEntry = document.createElement('div')
     taskEntry.className = 'imageTaskContainer'
     taskEntry.innerHTML = ` <div class="taskStatusLabel">Enqueued</div>
-                            <button class="secondaryButton stopTask"><i class="fa-solid fa-trash-can"></i> Remove</button>
+                            <div class="taskAction">
+                                <button class="secondaryButton useSettings" style="display: none"><i class="fa-solid fa-redo"></i>  Use Settings</button>
```

Comment:
Do we need the `display: none`? Is there ever a situation where this button should be hidden?

Reply:
Made the change. Use Settings is now always visible.